README.md

```diff
@@ -60,48 +60,44 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello_World] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- Good job using the Microverse README, but in other to make your project look more professional, please ensure that all all placeholders e.g `Describe your project in 1 or 2 sentences` have been removed from the README. Please fix this 🙏 
- Also, to help make your README more professional, please remove the Microverse logo, optionally, You can use your logo if you have any. Please fix this 🙏